### DISCUSSION TITLE
Are systems apps (and datas) backuped, even if not selected?

### DISCUSSION THREAD

**Discussion by guy-teube** [seedvault-app/seedvault#761] [Q&A]
Because I have a small space available in my Nextcloud account, I want to backup only some apps (and datas), not all. For my tests, I only tick 1 application (Signal) in the Backup status menu > [...] button.
But in the notification window, I see "system application", "animated wallpaper" and other stuffs which seem to be related to the system. See the attached screenshots for example.
As the backups are encrypted, I cannot verify what is really backuped.
So my question is: is there other files backuped, not selected by the user in the "exclude apps" menu? Is it possible to really see what is backuped, and really exclude system apps and others stuffs?

Bonus question: I think the "exclude menu" label is ambiguous: the ticked applications are included, or excluded? It seems only the ticked one are backuped, so the menu should be "Include apps" (means: I select those I want to backup). Maybe I have a understanding problem because of the language.

![Screenshot_20240930-184632_Seedvault](https://github.com/user-attachments/assets/0b2e1104-1b49-4dbb-858b-b1d8d4cde519)
![Screenshot_20240930-184626_Seedvault](https://github.com/user-attachments/assets/00ffa53d-c8c8-4ce0-af3d-d05ed57d8a7d)

**Comment by grote** [comment]
> For my tests, I only tick 1 application (Signal) in the Backup status menu

Take note that Signal doesn't use the system backup feature, but its own backup mechanism.

> in the notification window, I see "system application", "animated wallpaper" and other stuffs which seem to be related to the system.

You can't exclude these kind of system apps from backup, but they are really small.

> I think the "exclude menu" label is ambiguous: the ticked applications are included, or excluded? It seems only the ticked one are backuped

Correct. So you understood the meaning.

> so the menu should be "Include apps" 

All apps are included by default, so the use-case of the feature is to exclude apps.
